packages/teleport/src/Main/Main.tsx

```diff
@@ -41,14 +42,13 @@ import type { BannerType } from 'teleport/components/BannerList/BannerList';
 interface MainProps {
   initialAlerts?: ClusterAlert[];
   customBanners?: React.ReactNode[];
+  features: Feature[]; // TODO(ryan): move this into a context so we can use `useFeatures()` here
 }
 
 export function Main(props: MainProps) {
-  const [features] = useState(() => getFeatures());
```

Comment:
we will need this right? for the open source to get its features (unless i'm missing something) i'm fine if we did something like this for a quick patch (this was how i was doing it):

```ts
 const [features] = useState(() => {
    if (!props.customFeatures) {
      return getFeatures();
    }

    return props.customFeatures;
  });
```

Reply:
I've changed this to use a context now, and the correct value is passed in at the same time the Teleport context is created.